config/prow/cluster/secretrotator-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: default
+  name: secret-rotator
+  labels:
+    app: secret-rotator
+spec:
+  replicas: 1 # Do not scale up.
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: secret-rotator
+  template:
+    metadata:
+      labels:
+        app: secret-rotator
+    spec:
+      serviceAccountName: ks8-gsm-tools
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: secret-rotator
+        image: gcr.io/k8s-staging-k8s-gsm-tools/secret-rotator:latest
+        args:
+        - --config-path=/tmp/config/rotConfig 
+        - --period=1000
+        - --v=2
+        volumeMounts:
+        - name: config-volume
+          readOnly: true
+          mountPath: /tmp/config
+      volumes:
+      - name: config-volume
+        configMap:
+          name: config
```

Comment:
This ConfigMap is missing from the manifests here

You'll want it setup to rotate... uhh I know this isn't the exact file format, but hopefully this is enough for you to write it?
```
secret:
- namespace: test-pods
- name: service-account
- key: service-account.json
service-account:
- project: spiffxp-gke-dev
- name: spiffxp-gke-dev-prow
secretmanager:
- project: spiffxp-gke-dev
- secret: (whatever you want to call it)
```

Reply:
This secret should be the secret that allows prowjobs to write to GCS, and I have a job setup to run every 5min.  We'll be able to find out pretty quickly if rotation is working or not

Reply:
I pushed a configmap manifest that I think would work.
My question here was, if the job runs seamlessly, how are we going to distinguish whether the rotation is working fine or the service account key is simply not being rotated?